### PR TITLE
Update dmarcts-report-viewer.php

### PR DESCRIPTION
**Pull request by tachtler**
Change the sort order of the report results to DESCending. The newest entry will be shown on top.

### CODE REVIEW

**Comment by tachtler** [comment]
Maybe it's o.k. for you?

**Comment by lwt-pressy** [comment]
If we change the default behavior, we should do it by a config variable which goes by default (unset) to the old behavior for backward compability.

**Comment by tachtler** [comment]
What about my latest changes?

**Comment by tachtler** [comment]
I think latest will be better?

**Comment by lwt-pressy** [comment]
Your change by url will not work, after changing it messes up the url to dmarcts-report-viewer.php?hostlookup=1**&d=Array**

**Comment by tachtler** [comment]
You're right, now I'm at home - I will fix that.

**Comment by tachtler** [comment]
I'm sorry for that confusing code, but now can you give a try on the latest changes - at my installation it seems to work a expected?

**Comment by tachtler** [comment]
Latest changes.

File: dmarcts-report-viewer.php [hunk 1]
```diff
@@ -169,14 +169,15 @@ function tmpl_reportData($reportnumber, $allowed_reports, $host_lookup = 1) {
...
 	$url_hswitch = ( $reportid ? "?report=$reportid&hostlookup=" : "?hostlookup=" )
-                . ($host_lookup ? "0" : "1" )
+		. ($host_lookup ? "0" : "1" )
+                . ( "&sortorder=" ) . ($sort_order)
                 . (isset($dom_select) && $dom_select <> "" ? "&d=$dom_select" : "" )
                 ;
-	$url_dswitch = "?hostlookup=" . ($host_lookup ? "1" : "0" ); // drop selected report on domain switch
-
+        $url_dswitch = "?hostlookup=" . ($host_lookup ? "1" : "0" ) . "&sortorder=" . ($sort_order); // drop selected report on domain switch
+        $url_sswitch = "?hostlookup=" . ($host_lookup) . "&sortorder=" . ($sort_order ? "0" : "1" );
```
Comment (lwt-pressy) [inline dmarcts-report-viewer.php]: Here it is needed tp maintain the currently selected domain or a selected domain is lost. 
`        $url_sswitch = "?hostlookup=" . ($host_lookup) . "&sortorder=" . ($sort_order ? "0" : "1" ) . (isset($dom_select) && $dom_select <> "" ? "&d=$dom_select" : "" );`


This simple switching design works only nice with few variables someday we should do a recode with a function to generate the new url dynamicly.

Comment (lwt-pressy) [reply dmarcts-report-viewer.php]: also the README.md needs an update for the new config variable.

**Comment by tachtler** [comment]
Update README... but sorry I only have one Domain, so I can't test it right now.

**Comment by Nothing4You** [comment]
Could you sqash the commits please? It'll be less clutter in the commit history.

**Comment by tachtler** [comment]
I will open new Pull request.

**Pull request closed without merging**